libuvcConfig.cmake

```diff
@@ -0,0 +1,15 @@
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
+
+unset(extraArgs)
+if(${CMAKE_FIND_PACKAGE_NAME}_FIND_QUIETLY)
+  list(APPEND extraArgs QUIET)
+endif()
+if(${CMAKE_FIND_PACKAGE_NAME}_FIND_REQUIRED)
+  list(APPEND extraArgs REQUIRED)
+endif()
```

Comment:
This uses for forward `REQUIRED` and/or `QUITE` parameters to find_package below.